x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -404,6 +404,8 @@ const UnstyledProcessEventDot = React.memo(
               }}
               tabIndex={-1}
               title={eventModel.processNameSafeVersion(event)}
+              data-test-subj="resolver:node:primary-button"
```

Comment:
thoughts on this pattern?

Reply:
👍 I don't see a scenario where we add more buttons in the future, but if we do, then we can change it then. The possible future scenario I'm thinking about is something like this potentially around a process node: https://tympanus.net/codrops/2013/08/09/building-a-circular-navigation-with-css-transforms/

Probably a lot of accessibility concerns with that that I have not thought about though

Reply:
I like it